src/main/resources/META-INF/mule-artifact/mule-artifact.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Scheduler service",
-  "minMuleVersion": "4.0.0",
+  "minMuleVersion": "4.1.0",
```

Comment:
Should this be updated as part of this PR? Not an issue, but seems maybe some other build process maybe missing?

Reply:
the service requires the changes in mule-api becuase of this cahnge, so it it part of this.